addOns/reports/src/main/java/org/zaproxy/addon/reports/automation/ReportJobDialog.java

```diff
@@ -87,6 +89,9 @@ public class ReportJobDialog extends StandardFieldsDialog {
     private ExtensionReports extension = null;
     private ReportJob job;
 
+    private JScrollPane sitesPane;
+    private JList<String> sitesSelector;
```

Comment:
This does not allow to add/modify/remove sites (e.g. create a new plan with an empty session and you are not able to add the sites). There's also no support for variables (not a blocker I guess).